aiida/__init__.py

```diff
@@ -8,13 +8,14 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 import warnings
+from distutils.version import StrictVersion
```

Comment:
Do we want to have StrictVersion available at the `aiida` level?
Personally I prefer to hide it, so put it into the function.
But if you instead it's ok, I'm ok with a merge.

Reply:
I think that is a good point, I will move it in